include/vdml/vdml.h

```diff
@@ -17,6 +17,14 @@
 #include <stdint.h>
 #include "vdml/registry.h"
 
+#ifdef __cplusplus
+#define v5_device_e_t pros::c::v5_device_e_t
```

Comment:
This conditional define can probably get removed since we went with the `DeviceType` enum class instead of the `v5_device_e_t` enum.